charts/lagoon-remote/values.yaml

```diff
@@ -29,7 +29,7 @@ dockerHost:
 
   registry: registry.lagoon.svc:5000
 
-  repositoryToUpdate: lagoon
```

Comment:
I think this change might not work.

Looking at the script in https://github.com/amazeeio/lagoon/pull/2368 it greps for `${REPOSITORY_TO_UPDATE}/`. Our base image repositories are [`testlagoon`](https://hub.docker.com/u/testlagoon) and [`uselagoon`](https://hub.docker.com/u/uselagoon), so having `amazeeio` here will mean that the base images don't get updated. Could you confirm that understanding of where the base images are pushed @tobybellwood?

Reply:
It's an -E grep, so it probably should cover all our repos? 
`amazeeio|uselagoon` for production clusters, and add `testlagoon` if there's going to be test workloads on it?

`amazeeio|lagoon` would also cover it

Reply:
thanks for clarifying @tobybellwood 
in our production cluster, I saw `amazeeio` as base image repo.
If we agree I'd proceed using `amazeeio|lagoon`

Reply:
sounds good :+1: